eclcli/orchestration/heatclient/openstack/common/cliutils.py

```diff
@@ -19,11 +19,15 @@
 from __future__ import print_function
 
 import getpass
-import inspect
 import os
 import sys
 import textwrap
 
+try:
+    from inspect import getfullargspec as get_args
+except ImportError:
+    from inspect import getargspec as get_args
+
 from oslo_utils import encodeutils
```

Comment:
I think that it is necessary to describe like other processing.
```suggestion
try:
    from oslo_utils import encodeutils
except ImportError:
    from oslo.utils import encodeutils
```